packages/design-system-tokens/src/themes/healthcare-components.ts

```diff
@@ -0,0 +1,254 @@
+/*
+ * Healthcare.gov CMSDS Component Theme
+ */
+
+import { default as t } from './healthcare'
```

Comment:
At first I was unsure about having this separate file that is directly handled by the exporter instead of importing this components file into the base theme file and spreading it on the exported theme object. But now I realize how much nicer this is because the components file can _consume_ the base tokens for the theme. It's not like any other tokens will need access to the tokens defined in this components file, so this flow of information makes sense.